EpiserverRedirects/ClientResources/RedirectsMenu/RedirectsMenuGrid/RedirectsMenuGrid.css

```diff
@@ -17,11 +17,12 @@
 }
 
 .url-redirect-grid-container .dgrid .dgrid-scroller {
-    margin-top: 55px;
+    margin-top: 100px !important;
+    margin-bottom: 30px !important;
 }
 
 .url-redirect-grid-container .dgrid .dgrid-header {
-    height: 55px;
+    height: 100px !important;
```

Comment:
![image](https://user-images.githubusercontent.com/12833315/163404724-24535757-786c-4d8b-ab1a-3d290bbf72b0.png)

Reply:
Unfortunately, the header receives inline styles when sorting by column so it was kind of the cheapest solution...
I really, really regret causing this death